doc/src/site/sphinx/tutorials/hive_support.rst

```diff
@@ -69,9 +65,100 @@ from in the following ways.
 - You can also retrieve this from Maven for the desired version using
   ``mvn dependency:get -Dartifact=groupId:artifactId:version``.
 
-Generate Initial Token
-^^^^^^^^^^^^^^^^^^^^^^
-The initial delegation token can be generated with the following steps.
+Import Data from Hive in a non-Kerberized Hadoop cluster
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+To import data from non-Kerberized Hadoop cluster, run:
+
+.. content-tabs::
+
+    .. tab-container:: Scala
+        :title: Scala
+
+        First, start Sparkling Shell with the  Hive JDBC client jar on the class path
+
+        .. code:: bash
+
+            ./bin/sparkling-shell --jars /path/to/hive-jdbc-<version>-standalone.jar
+
+        Create ``H2OContext`` with properties ensuring connectivity to Hive
+
+        .. code:: scala
+
+            import ai.h2o.sparkling._
+            val conf = new H2OConf()
+            conf.setHiveSupportEnabled()
```

Comment:
Have you tried to run code snippets?  if I'm not mistaken and `setHiveSupportEnabled()` is called, a delegation token is required for refresh.

Reply:
Nope, just updated the docs. 
I originally thought that you need token only in case of Kerberos for auth

Reply:
Just checked with @honzasterba. We do not need delegation token in case when we are not talking to kerberized Hive

Reply:
[This precedure](https://github.com/h2oai/sparkling-water/blob/82885bd5611a4a26380c2d4431415058bc2596bc/core/src/main/scala/org/apache/spark/h2o/backends/internal/InternalH2OBackend.scala#L174) will be executed on every executor, are you sure it won't fail in non-kerberized cluster?

Reply:
@honzasterba As long as I understand it, I think it should work fine, but I'm Hive noob. 
cc: @honzasterba

Reply:
yes, DelegationTokenRefresher is only necessary when hive is kerberized

Reply:
thanks for confirmation!

Reply:
perhaps the setHiveSupportEnabled function name is confusing?

Reply:
Yup, that is a good point. It actually allows path only for kerberized hive, normal hive can work without it

Reply:
> perhaps the setHiveSupportEnabled function name is confusing?

+1 This property should be renamed.

Reply:
https://0xdata.atlassian.net/browse/SW-2268 I think we can rename and deprecate the old option also for release 3.30 and remove in 3.32